README.md

```diff
@@ -106,6 +108,36 @@ namespace SomeApi
 
 ```
 
+#### Logging scope middleware
+The logging scope middleware sets up a logging scope for every incoming HTTP request. 
+This means that every log statement made within that scope (i.e. during the HTTP request processing) 
+will include an addition string that contains both the correlation id (and user id, if available 

```

Comment:
I understand that if the correlation id is not available then the correlationIdMiddleware will generate one, is that the same for userid?

Reply:
At them moment we still haven't decided much about how the user id will be identified on a call, but my assumption is that it will always be there. That said if there isn't a value for either of them then it won't break, the log statements just won't have them included.